Mathlib/MeasureTheory/Measure/RegularityCompacts.lean

```diff
@@ -11,13 +11,14 @@ import Mathlib.Topology.UniformSpace.Cauchy
 /-!
 # Inner regularity of finite measures
 
-The main result of this file is `theorem InnerRegularCompactLTTop`:
+The main result of this file is
+`InnerRegularCompactLTTop_of_pseudoEMetricSpace_completeSpace_secondCountable`:
 A finite measure `μ` on a `PseudoEMetricSpace E` and `CompleteSpace E` with
 `SecondCountableTopology E` is inner regular with respect to compact sets. In other
 words, a finite measure on such a space is a tight measure.
 
 Finite measures on Polish spaces are an important special case, which makes the result
-`theorem PolishSpace.innerRegular_isCompact_measurableSet` an important result in probability.
+`PolishSpace.innerRegular_isCompact_isClosed_measurableSet` an important result in probability.
```

Comment:
This looks fine to me, although it seems that the full name is in the `MeasureTheory` namespace.

Reply:
Huh, so it is. I don't think it should be, but won't change that here